Frequency/Board/Map/Tile.py

```diff
@@ -16,14 +15,20 @@ def __init__(self, position, defaultMoney, enemyMoney, texture, size, units=None
            self.Units = []
         else:
             self.Units = units
+        self.Building = building
 
 
     def Update(self, game):
         if self.IsClickedByMouse(game):
-            print(self.Position.X, self.Position.Y)
-            self.Units.append(Soldier('Player1', self))
+            if game.Settings.GetSelectedUnitBuilding() == "Soldier":
```

Comment:
method is mij niet direct duidelijk wat het zou doen en hier mag je het settings object echt niet voor misbruiken